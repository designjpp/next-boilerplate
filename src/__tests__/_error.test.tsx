import { render } from '@testing-library/react';
import { axe } from 'jest-axe';
import { create } from 'react-test-renderer';

import ErrorPage from 'pages/_error';

describe('<ErrorPage>', () => {
	it('should match snapshot', async () => {
		const component = create(<ErrorPage />).toJSON();
		expect(component).toMatchSnapshot();
	});

	it('should be accessible', async () => {
		const { container } = render(<ErrorPage />);
		const results = await axe(container);
		expect(results).toHaveNoViolations();
	});
});
