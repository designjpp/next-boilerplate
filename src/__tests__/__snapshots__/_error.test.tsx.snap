// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<ErrorPage> should match snapshot 1`] = `
<p>
  An error occurred on client
</p>
`;
